ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ultrassd-disk/cucushift-installer-rehearse-azure-ipi-ultrassd-disk-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-ultrassd-disk
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-azure-ipi-ultrassd-disk-provision
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    post:
+    - chain: cucushift-installer-rehearse-azure-ipi-ultrassd-disk-deprovision
```

Comment:
Based on [comments](https://github.com/openshift/release/pull/38488/files#r1172038605), I think we don't need to introduce the new chain `cucushift-installer-rehearse-azure-ipi-ultrassd-disk-deprovision`, but just only reuse the chain `cucushift-installer-rehearse-azure-ipi-deprovision`

Reply:
thanks @jinyunma , made changes